examples/curations.yml

```diff
@@ -59,3 +59,9 @@
     comment: "The package has an unmappable declared license entry."
     declared_license_mapping:
       "BSD-derived (http://www.repoze.org/LICENSE.txt)": "LicenseRef-scancode-repoze"
+
+- id: "PyPI::branca"
```

Comment:
Unrelated, but it just occurred to me that while I believe "PyPI" is correct here, shouldn't it also be "PyPI" instead of "Pip" in line 57? If so, mind adding a commit to your PR?

Reply:
Right, done.